js/navigation.js

```diff
@@ -162,37 +162,38 @@ class NavBar {
 				title: "Add the site to your home screen. When used in conjunction with the Preload Offline Data option, this can create a functional offline copy of the site.",
 			},
 		);
-		this._addElement_label(NavBar._CAT_SETTINGS,
+		this._addElement_dropdown(NavBar._CAT_SETTINGS, NavBar._CAT_CACHE, {isSide: true});
```

Comment:
Can we make this ``{isSide: false}`` so that there is not overflow on many devices? Otherwise LGTM